test/components/notification.spec.js

```diff
@@ -1,203 +1,191 @@
-import $ from 'jquery'
 import Notification from '../../src/js/components/notification'
 import transitionEnd from '../../src/js/utils/transitionend'
+import triggerEvent from '../../src/js/utils/trigger-event'
 
 describe('Notification spec', () => {
-  let instance, $fixture, options
+  let instance, fixtureElement, options
 
   before(() => {
     fixture.setBase('test/fixture')
   })
 
   beforeEach(() => {
-    $fixture = $(fixture.load('notification.html')[0])
+    fixtureElement = fixture.load('notification.html')[0]
 
     options = {
       message: 'foo'
     }
 
-    instance = new Notification($fixture.find('[data-notification]'), options)
+    instance = new Notification(fixtureElement, options)
   })
 
   afterEach(() => {
     fixture.cleanup()
   })
 
-  describe('consutructor', () => {
-    it('should return a instanceof $ if param element isn\'t', () => {
-      let newIstance = new Notification(fixture.load('notification.html')[0], options)
-
-      expect(newIstance.$element).to.be.instanceof($)
-    })
-  })
-
   describe('init', () => {
     it('should create notification in DOM', sinon.test(function () {
-      let spy = this.spy(instance, '_createNotification')
+      const spy = this.spy(instance, 'createNotification')
 
       instance.init()
 
-      expect(spy.calledOnce).to.be.true
+      expect(spy.calledOnce).to.true
```

Comment:
`to.be.true` é mais legível

Reply:
Eu só tirei pra manter a consistencia, ta tudo trocado, se for para ter `be` em tudo acho melhor